samples/nrf_desktop/src/events/battery_event.c

```diff
@@ -26,10 +28,10 @@ static void print_battery_state_event(const struct event_header *eh)
 
 	__ASSERT_NO_MSG(event->state < BATTERY_STATE_COUNT);
 
-	printk("battery %s", state_name[event->state]);
+	return snprintf(buf, buf_len, "battery %s", state_name[event->state]);
 }
 
-static void log_battery_state_event(struct log_event_buf *buf,
+static void profiler_log_battery_state_event_args(struct log_event_buf *buf,
```

Comment:
if you use `log_battery_state_event()` for logging now, what about `profile_battery_state_event()` for profiler?

Reply:
done